README.md

```diff
@@ -1,5 +1,19 @@
 ### Arrays
 
+#### First
+
+* Underscore
+  ```javascript
+  _.first([5, 4, 3, 2, 1]);
+  ```
+
+* ES2015
+
+  ```javascript
+  const first = ([n]) => n;
+  first([5, 4, 3, 2, 1]);
```

Comment:
I'm still not sure about this. I wouldn't define a `first` function, because it might be misleading – the point is that you might not need a `first` function.

If the idea here is to show how you can use destructuring, I think maybe we could do this:
```javascript
var first = _.first(array);
```
and
```javascript
var [first] = array; 
```